db/controller.js

```diff
@@ -71,3 +72,12 @@ exports.addVote = (req, res, next) => {
     })
     .catch(next);
 };
+
+exports.deleteComment = (req, res, next) => {
+  const { comment_id } = req.params;
+  deleteCommentById(comment_id)
+    .then((comment) => {
+      res.status(204).send({ comment });
+    })
```

Comment:
nice - we don't actually need to even send anything back here we could just send the status of 204 :)